### ISSUE TITLE
Using pkgconfig

### ISSUE THREAD

**Issue by cburstedde** [DLR-AMR/t8code#500]
I'm wondering whether anybody has experience generating and using pkgconfig files.
libsc has a mechanism to create one, but it is not nearly sufficiently tested.

pgkconfig allows to specify separate paths for lib, include, etc, corresponding to using separate --prefix type configure options. Currently when we use --with-sc this expects all installed files in the same tree, which is not how it works for typical linux distributions. Thus, with --with-sc, we may need to add --with-sc-lib, --with-sc-include, etc. to make it practical.

Bottom line: I'm happy to help working on the pkgconfig system of libsc (und thus p4est and t8code) but I'd need some help by someone who has sufficient practical experience with this concept and workflow. Volunteers?

**Comment by holke** [comment]
We do not have experience either, but @ililikakis will allocate some time to look into this.

**Comment by ole-alb** [comment]
We currently don't have any capacity to work on that. If that changes we might reopen this Issue.
